### DISCUSSION TITLE
Vertical Slider

### DISCUSSION THREAD

**Discussion by PointSingularity** [mantinedev/mantine#1361] [Q&A] [answered]
Is it possible to modify the current Slider component to be vertical?

My plan was implementing a volume slider. I have tried to do it, and with using styles to transform the orientation it can look right. The problem is that  dragging the thumb with the mouse reacts only when moving the mouse in the horizontal direction.

Any help would be greatly appreciated.

**Comment by rtivital** [comment, marked as answer]
https://mantine.dev/hooks/use-move/#vertical-slider

**Reply by PointSingularity** [reply]
I can't believe I missed that (it doesn't show on the spotlight (Ctrl + k) search).

Thank you for answering 👍
